### DISCUSSION TITLE
[Feature Req] Allowlist certain specific types or kinds of requests

### DISCUSSION THREAD

**Discussion by npub1zenn0** [diegogurpegui/nos2x-fox#51] [Ideas]
Using this extension, one can only auto-permit all requests or nothing. However, for requests like `AUTH`, it would be great to allow them for a certain website always, because they have to be remade for each connection. But I feel less confident in allowing signing anything, mostly in case of bugs that'd cause my feed to explode.

**Comment by diegogurpegui** [comment]
That makes sense!
